backend/db/resolve_fetch_failures_0_members.py

```diff
@@ -162,22 +162,29 @@ def resolve_fetch_failures_0_members(
     if still_draft_cset_ids:
         print(f"Fetch attempted for the following csets, but they still remain drafts and thus still have not had their"
               f" members expanded yet: {', '.join([str(x) for x in still_draft_cset_ids])}")
-    true_failed_cset_ids: Set[int] = set(failed_cset_ids) - still_draft_cset_ids
+    non_draft_failure_ids: Set[int] = set(failed_cset_ids) - still_draft_cset_ids
+
+    # Filter by only if has been finalized longer than we would expect it should take for expansion to be available
+    # noinspection PyUnboundLocalVariable
+    still_draft_csets: List[Dict] = [
+        x['properties'] for x in csets_and_members['OMOPConceptSet']
+        if x['properties']['codesetId'] in non_draft_failure_ids]
+    final_failure_ids: Set[int] = get_csets_over_threshold(still_draft_csets, int(expansion_threshold_seconds / 60))
```

Comment:
Now errors only if the cset is too old. We figure that 2 hours is a sufficient amount of time to give to the Enclave to expand.

@Sigfried Should we take this time not simply to throw an error, but should we mark as finalized if this happens?
I have a better proposal, though:
`TODO: if every expression is set to `isExcluded=true`, we expect no members. Can add as a check rather than wait 2hrs`